src/20_Components/amp-bind.html

```diff
@@ -65,6 +65,7 @@
       You can dynamically change the CSS classes of an element by adding a binding to the `[class]` attribute.
     -->
     <div>
+      <amp-state id="myClass"><script type="application/json">""</script></amp-state>
```

Comment:
We need to restructure the sample, as we haven't introduced the amp-state element at this point.

Reply:
Should I move this section after the part that introduces `amp-state`?

Reply:
How about moving the initialising state sample after binding text?

Reply:
done